capstone-frontend/package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "jquery": "^3.5.1",
```

Comment:
Did you end up using jquery for this PR? I thought I remember you talking about removing it and replacing it with Javascript fetches

Reply:
I didn't, but removed it in the next PR off of this one (https://github.com/STEP-Pod-Capstone-Project/Capstone-Project/pull/26). I can just remove it in this one.